src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
+    // Constants
+    private static final String MESSAGE_BOUNDARY = "____________________________________________________________";
+
+    // Attributes
+    private static Task[] tasks = new Task[100];
+    private static int numoftask = 0;
+
+    // Constructor
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Greet();
+        Echo();
     }
+
+    public static void Greet(){
+        String lines = "____________________________________________________________";
+        String greet = " Hello! I'm Duke\n"
+                    + " What can I do for you?\n";
+        System.out.println(MESSAGE_BOUNDARY + "\n" + greet + MESSAGE_BOUNDARY);
+    }
+
+    public static void Echo(){
+        boolean continueChat;
+        Scanner in = new Scanner(System.in);
+
+        do{
+            // Get query from user
+            String userInput = in.nextLine();
+
+            // Give response
+            continueChat = giveResponse(userInput);
+
+        }while(continueChat);
+
+        in.close();
+    }
+
+    public static boolean giveResponse(String userInput){
+        if(userInput.equals("list")){
+            printList();
+        }else if(userInput.equals("bye")){
+            Exit();
+            return false;
+        }else{
+            int i = userInput.indexOf(" ");
+            String command = userInput.substring(0, i);
+
+            switch (command){
+                case "todo":
+                case "deadline":
+                case "event":
+                    addTask(command, userInput.substring(i + 1));
+                    break;
+                case "done":
+                    doneTask(userInput.substring(i + 1));
+                    break;
+                default:
+                    System.out.println("I am sorry, but I don't understand.");
+            }
+        }
+
+        return true;
+    }
+
+    public static void Exit(){
+        String bye = " Bye. Hope to see you again soon!\n";
+        System.out.println(MESSAGE_BOUNDARY + "\n" + bye + MESSAGE_BOUNDARY);
+    }
+
+    public static void doneTask(String index){
```

Comment:
Maybe rename this to explain what this method does with an action verb (e.g. markTaskAsDone)? As per the coding standard, "names representing methods must be verbs and written in camelCase" (https://se-education.org/guides/conventions/java/basic.html#naming).